lib/curator.rb

```diff
@@ -0,0 +1,58 @@
+class Curator
+  attr_reader :photographs, :artists
+
+  def initialize
+    @photographs = []
+    @artists = []
+  end
+
+  def add_photograph(photo)
+    @photographs << photo
+  end
+
+  def add_artist(artist)
+    @artists << artist
+  end
+
+  def find_artist_by_id(artist)
+    artists.each do |artist|
```

Comment:
Each returns the original array - in order to get this method working, you would need to use `.find` or create an accumulator to grab the artist.